pkg/scheduler/cache/cluster_info/cluster_info.go

```diff
@@ -449,6 +455,15 @@ func getPodGroupPriority(
 	return chosenPriorityClass.Value
 }
 
+func getPodGroupPreemptibility(preemptibility enginev2alpha2.Preemptibility, priority int32) enginev2alpha2.Preemptibility {
```

Comment:
This function feels redundant - it just ignores the error, you can do that in `snapshotPodGroups` as well